pkg/login/social/github_oauth.go

```diff
@@ -197,7 +197,7 @@ func (s *SocialGithub) UserInfo(client *http.Client, token *oauth2.Token) (*Basi
 
 	role, grafanaAdmin := s.extractRoleAndAdmin(response.Body, teams, true)
 	if s.roleAttributeStrict && !role.IsValid() {
-		return nil, &InvalidBasicRoleError{idP: "Github", assignedRole: string(role)}
+		return nil, &InvalidBasicRoleError{idP: "GitHub", assignedRole: string(role)}
```

Comment:
@grafana/grafana-authnz-team Are you able to review this change? It looks like this property is only used for an error message (and it's a meaningful string/'enum'), but would like to be extra sure :)

Reply:
Ahh, sorry about that.
Just let me know if it needs to be undone.

With bulk changes, I usually only target docs/html/comments/prints/etc, so this one was a mistake.

Reply:
Yes this is only used for error message so the change is fine :)